plugins/org.jkiss.dbeaver.ext.generic/plugin.xml

```diff
@@ -1423,7 +1423,7 @@
                         webURL="https://www.mysql.com/products/connector/"
                         supportedConfigurationTypes="MANUAL,URL"
                         description="Driver for MySQL (4.x and earlier)">
-                    <file type="jar" path="maven:/mysql:mysql-connector-java:RELEASE"/>
+                    <file type="jar" path="maven:/com.mysql:mysql-connector-j:RELEASE"/>
```

Comment:
```suggestion
                    <file type="jar" path="maven:/com.mysql:mysql-connector-j:RELEASE[8.2.0]"/>
```

Reply:
Please add a specific version. We can not be sure that the next release will not have vulnerabilities.